src/components/QueryAt/index.js

```diff
@@ -45,16 +45,15 @@ class QueryAtImplementation extends Component {
                 Prismic.Predicates.at(path, value),
             );
         }).then((response) => {
-            this.setState({ response });
+            this.setState({ results: response.results });
```

Comment:
there are info in the response you don't want to lose .. its better to send the entire response